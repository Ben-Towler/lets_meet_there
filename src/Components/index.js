import SiteHeader from './SiteHeader/SiteHeader.component';
import SearchForm from './SearchForm/SearchForm.component';
import Trip from './Trip/Trip.component';
import Button from './Button/Button.component';
import Form from './Form/Form.component';
import Flight from './Flight/Flight.component';
import FlightSummary from './FlightSummary/FlightSummary.component';
import BookingLink from './BookingLink/BookingLink.component';
import Spinner from './Spinner/Spinner.component';
import LocationSearch from './LocationSearch/locationSearch.component.tsx';

export {
  SiteHeader,
  BookingLink,
  SearchForm,
  Trip,
  Flight,
  FlightSummary,
  Button,
  Form,
  Spinner,
  LocationSearch
}