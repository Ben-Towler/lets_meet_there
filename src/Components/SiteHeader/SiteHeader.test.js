import React from 'react';
import { SiteHeader } from 'Components';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('Site Header', () => {

  test('snapshot matches', () => {
    const renderer = new ShallowRenderer()
    const result = renderer.render(
      <SiteHeader 
        user={null} 
      />
    )
    expect(result).toMatchSnapshot()
  });

  test('snapshot matches', () => {
    const renderer = new ShallowRenderer()
    const result = renderer.render(
      <SiteHeader 
        user={true} 
      />
    )
    expect(result).toMatchSnapshot()
  });

});

