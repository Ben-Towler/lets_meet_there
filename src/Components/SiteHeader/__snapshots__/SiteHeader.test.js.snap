// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site Header snapshot matches 1`] = `
<ForwardRef(styled.header)>
  <ForwardRef
    to="/"
  >
    Home
  </ForwardRef>
  <Button
    onClick={[Function]}
  >
    Sign In with Google
  </Button>
</ForwardRef(styled.header)>
`;

exports[`Site Header snapshot matches 2`] = `
<ForwardRef(styled.header)>
  <ForwardRef
    to="/"
  >
    Home
  </ForwardRef>
  <React.Fragment>
    <Button>
      Sign Out
    </Button>
    <ForwardRef
      to="/profile"
    >
      Profile
    </ForwardRef>
  </React.Fragment>
</ForwardRef(styled.header)>
`;
